### PR TITLE
Load AIE only PDI

### PR DESCRIPTION
**Pull request by larry9523**

### CODE REVIEW

File: src/runtime_src/core/edge/user/aie/graph.cpp [hunk 1]
```diff
@@ -538,6 +538,14 @@ xrtSyncBOAIE(xrtGraphHandle ghdl, unsigned int bo, const char *dmaID, enum xclBO
   graph->sync_bo(bo, dmaID, dir, size, offset);
 }
 
+void
+xrtResetAieArray(xclDeviceHandle handle)
+{
+  auto device = xrt_core::get_userpf_device(handle);
+  XAieLib_NpiAieArrayReset(XAIE_RESETENABLE);
+  XAieLib_NpiAieArrayReset(XAIE_RESETDISABLE);
```
Comment (stsoe) [inline src/runtime_src/core/edge/user/aie/graph.cpp:546]: Doesn't look the device is used here?  Can you remove line 544?

Comment (larry9523) [reply src/runtime_src/core/edge/user/aie/graph.cpp:546]: My intention here is to do a handle check. The caller should open the device before operate on it. Do you think it is necessary? Or do we have a better way to do it?

Comment (stsoe) [reply src/runtime_src/core/edge/user/aie/graph.cpp:546]: Ok. intention is fine, makes sense.  Maybe comment to that effect and maybe don't save the return value.

Comment (larry9523) [reply src/runtime_src/core/edge/user/aie/graph.cpp]: Done. Thanks!

**Comment by larry9523** [comment]
This PR depends on https://github.com/Xilinx/XRT/pull/3479
I will remove the "do not merge" label after the above PR is merged

**Pull request merged**